app/controllers/users/reset_passwords_controller.rb

```diff
@@ -94,7 +94,7 @@ def handle_successful_password_reset
     end
 
     def handle_unsuccessful_password_reset(result)
-      if result.errors[:reset_password_token]
+      if result.errors[:reset_password_token].present?
```

Comment:
Was this related to the Rails upgrade in some way, or just catching a best practice?